docs/1.getting-started/8.error-handling.md

```diff
@@ -86,8 +86,10 @@ Customize the default error page by adding `~/error.vue` in the source directory
 
 ```vue [error.vue]
 <script setup lang="ts">
+import type { NuxtError } from '#app'
+
 const props = defineProps({
-  error: Object
+  error: NuxtError
```

Comment:
This should be `Object as () => NuxtError` here and below.

Reply:
Good point. Do we want to switch to the following though?
```
const props = defineProps<{
  error: NuxtError
}>()
```

Reply:
No, I think not. It will require vue to have typescript installed as a peerDependency, and doesn't add any additional validation or type safety